docs/algorithms/sig/falcon.yml

```diff
@@ -17,7 +18,7 @@ website: https://falcon-sign.info
 nist-round: 3
 spec-version: v1.2
 primary-upstream:
-  source: https://github.com/PQClean/PQClean/commit/33bceb17eb06a40fbdc72251f533734e8d869615
+  source: https://github.com/PQClean/PQClean/commit/245c95cd1ec326f8f38e26cb17a28832701ba17b
   spdx-license-identifier: CC0-1.0
   upstream-ancestors:
```

Comment:
Is this license identifier still correct given the newly documented patent "caveat"?

Reply:
This license identifier was never correct, as Falcon's implementation is MIT-licensed.

Reply:
:-) :-/ An opportunity to correct this then, no?